backend/src/ee/services/permission/org-permission.ts

```diff
@@ -28,6 +28,14 @@ export enum OrgPermissionKmipActions {
   Setup = "setup"
 }
 
+export enum OrgPermissionMachineIdentityAuthTemplateActions {
```

Comment:
Please add the permissions here

<img width="1512" height="831" alt="image" src="https://github.com/user-attachments/assets/cd3c9ba7-85d2-43d9-b153-13d180c685c1" />